feature/deps.go

```diff
@@ -75,6 +75,10 @@ var deps = depDesc{
 	lnwire.ZeroConfOptional: {
 		lnwire.ScidAliasOptional: {},
 	},
+	lnwire.SimpleTaprootChannelsOptional: {
```

Comment:
should we add a `cfg.NoTaproot` option which can then be used in `feature.newManager` to unset this bit?

We also need to unset it there (in `newManager`) if any of the dependent bits have been unset (ex using `cfg.NoAnchors)

Reply:
Can add this in part 8 as that's where the set of command line options are exposed for the first time.